pyproject.toml

```diff
@@ -1,13 +1,14 @@
 [tool.poetry]
 name = "data-platform-helpers"
-version = "0.1.3"
+version = "0.1.4"
 description = ""
 authors = ["Mia Altieri <mgaltier200@gmail.com>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.10"
 ops = "^2.15.0"
+tenacity = "^9.0.0"
```

Comment:
mysql has a tenacity dependency of `^8.2.2`. should we be specifying versions in our dependencies at all?

Reply:
i got test running without specifying any dependencies but `python ^3.10` in `pyproject.toml`. specifying any more dependencies leads to conflics/overwrites of pinned dependencies